test/test.cpp

```diff
@@ -274,17 +274,17 @@ bool runClientTests(bool standalone, bool verbose)
 // Standalone tests don't really use WSD
 #ifndef STANDALONE_CPPUNIT
 
-std::vector<int> getKitPids()
+std::set<pid_t> getKitPids()
 {
     return LOOLWSD::getKitPids();
 }
 
 /// Get the PID of the forkit
-std::vector<int> getForKitPids()
+std::set<pid_t> getForKitPids()
```

Comment:
Interesting. We can ever have a single ForKit instance, by design. Do we really need to create and return a set here? (Not sure why we had a vector in the first place.)

Looks like a nice cleanup to do, I think, to simply return a single PID. Perhaps can be done separately, and not in this PR. Up to you.

Reply:
I've already seen this but decided not to change it. It returns a set (previously vector) probably because its return value is passed to killPids: https://github.com/CollaboraOnline/online/blob/de62c0f359ecb589d8480889092c0b6592fe56ed/test/httpcrashtest.cpp#L252